app/views/verify/authentications/failed.html.erb

```diff
@@ -2,22 +2,35 @@
   <div class="govuk-grid-column-two-thirds">
 
     <h1 class="govuk-heading-xl">
-      Your identity could not be verified
+      We could not verify your identity
     </h1>
 
     <p class="govuk-body">
-      There are a few reasons why this might happen, including:
+      This might be because:
     </p>
 
     <ul class="govuk-list govuk-list--bullet">
       <li>
-        your answer to a question doesn't match the records held by the Verify provider
+        the company you chose does not have enough information about you
       </li>
       <li>
-        the Verify provider that you chose does not have access to enough information about you
+        the company’s records are not up to date
```

Comment:
Is it clear that `the company` referred to here refers to `the company you chose`, or should we repeat this?

Reply:
We call it the company you chose up above this, so though I considered it, repeating the 'you chose' seemed a bit excessive to me.

Reply:
Cool fair enough 👍